packages/wrangler/src/logger.ts

```diff
@@ -102,6 +102,11 @@ export class Logger {
 
 	columns = process.stdout.columns;
 
+	json = (data: unknown) => {
+		// eslint-disable-next-line no-console
+		console.log(JSON.stringify(data, null, 4));
```

Comment:
Shouldn't this use `logger.log()`? Otherwise it won't respect the configured log level

Reply:
We can't use `logger.log()` because it may add extra "stuff" that would stop the output from being valid JSON.